airflow/contrib/hooks/gcp_dlp_hook.py

```diff
@@ -0,0 +1,1760 @@
+
+        return client.create_stored_info_type(
+            parent=parent,
+            config=config,
+            stored_info_type_id=stored_info_type_id,
+            retry=retry,
+            timeout=timeout,
+            metadata=metadata,
+        )
+
+    @GoogleCloudBaseHook.catch_http_exception
+    @GoogleCloudBaseHook.fallback_to_default_project_id
+    def deidentify_content(
+        self,
+        project_id=None,
+        deidentify_config=None,
+        inspect_config=None,
+        item=None,
+        inspect_template_name=None,
+        deidentify_template_name=None,
+        retry=None,
+        timeout=None,
+        metadata=None,
+    ):
+        """
+        De-identifies potentially sensitive info from a content item. This method has limits
+        on input size and output size.
+
+        :param project_id: (Optional) Google Cloud Platform project ID where the
+            DLP Instance exists. If set to None or missing, the default
+            project_id from the GCP connection is used.
+        :type project_id: str
+        :param deidentify_config: (Optional) Configuration for the de-identification of the
+            content item. Items specified here will override the template referenced by the
+            deidentify_template_name argument.
+        :type deidentify_config: dict or google.cloud.dlp_v2.types.DeidentifyConfig
+        :param inspect_config: (Optional) Configuration for the inspector. Items specified
+            here will override the template referenced by the inspect_template_name argument.
+        :type inspect_config: dict or google.cloud.dlp_v2.types.InspectConfig
+        :param item: (Optional) The item to de-identify. Will be treated as text.
+        :type item: dict or google.cloud.dlp_v2.types.ContentItem
+        :param inspect_template_name: (Optional) Optional template to use. Any configuration
+            directly specified in inspect_config will override those set in the template.
+        :type inspect_template_name: str
+        :param deidentify_template_name: (Optional) Optional template to use. Any
+            configuration directly specified in deidentify_config will override those set
+            in the template.
+        :type deidentify_template_name: str
+        :param retry: (Optional) A retry object used to retry requests.
+            If None is specified, requests will not be retried.
+        :type retry: google.api_core.retry.Retry
+        :param timeout: (Optional) The amount of time, in seconds, to wait for the request
+            to complete. Note that if retry is specified, the timeout applies to each
+            individual attempt.
+        :type timeout: float
+        :param metadata: (Optional) Additional metadata that is provided to the method.
+        :type metadata: sequence[tuple[str, str]]]
+        :rtype: google.cloud.dlp_v2.types.DeidentifyContentResponse
+        """
+
+        client = self.get_conn()
+
+        if not project_id:
```

Comment:
It looks like dead code, because decorator ``fallback_to_default_project_id`` prevent to execute this part of code